test/ssl-tests/20-cert-select.cnf.in

```diff
@@ -14,6 +14,18 @@ our $fips_3_4;
 our $fips_3_5;
 our $no_deflt_libctx;
 
+srand(20);
```

Comment:
And this is a much better way to test than editing test/ssl-tests/20-cert-select.cnf in my view

Reply:
As noted, I did not "edit" the generated files. I ran the generator, but the output needs to be stable for reasons I don't fully know, but guessing it is to avoid accidental failure to test when the generator produces no tests in its output.